dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -84,7 +85,80 @@ public Pattern getParameterPattern() {
 
     public Map<String, DbParameterAccessor> getAllProcedureParameters(
             String procName) throws SQLException {
-        throw new UnsupportedOperationException();
+        String[] qualifiers = procName.split("\\.");
```

Comment:
I guess the trade off of moving some logic out to other methods is that some people will find reading though the method hard because of the need to jump about to different methods instead of having a continuous flow.

I'll see what it looks like with some parts moved out.